nn_sdwan/assets/monitors/high-latency-monitor.json

```diff
@@ -0,0 +1,23 @@
+{
+    "id": 65951821,
```

Comment:
Not sure what the `id` is, this should be deleted

Reply:
Oh I think this is because I created the monitor under my account, then exported a fully created/registered monitor. So there are fields related to my account/org/permissions/etc. Exporting the monitor before officially creating it in my account seems to fix this.